### PR TITLE
custom inv, rel note, eval

### PR DESCRIPTION
**Pull request by MariePMcGarry**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by kimsauce — CHANGES_REQUESTED** [review]
* In **docs/cse/administration/custom-inventory-sources.md**, the header "### Webhook payload for Computer Entity" is displayed as code. Code snippet above that is missing the bottom backticks.
* In the same file, italics is broken in this sentence: The type key specifies what type of inventory data the webhook sends. Set the value to user or _computer. _.

File: docs/cse/administration/custom-inventory-sources.md [hunk 1]
```diff
@@ -294,10 +287,11 @@ _`sourceCategory="security/jamf" and _collector="Jamf" \
 "customInventory": true,
 "type": "user"
 }
-
+```
 
 ### Webhook payload for Computer Entity
 
+```
```
Comment (kimsauce) [inline docs/cse/administration/custom-inventory-sources.md:294]: Code below and above are JSON, so I would enable syntax highlighting for them. 

```suggestion
```json
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor changes

File: docs/cse/administration/custom-inventory-sources.md [hunk 2]
```diff
@@ -259,23 +252,23 @@ The table below lists attributes most typically used in user inventory records.
...
+| json field _raw "event.computer.osVersion as os_version <br/>
+| json field _raw "event.computer.deviceName as hostname  <br/>
+| json field _raw "event.computer.deviceName as hostname  <br/>
+| json field _raw "event.computer.ipAddress as ip  <br/>
+| json field _raw "event.computer.macAddress as mac  <br/>
+| json field _raw "event.computer.username as username  <br/>
+| json field _raw "event.computer.emailAddress as email  <br/>
+| json field _raw "event.computer.position as role  <br/>
+| where !(isEmpty(username))  <br/>
+| count by os_version, hostname, ip, mac, username,email, role</code>
```
[the remark this answers is not in the document]
Comment (MariePMcGarry) [reply docs/cse/administration/custom-inventory-sources.md]: Updated file in PR

[the remark this answers is not in the document]
Comment (MariePMcGarry) [reply docs/cse/administration/custom-inventory-sources.md]: fixed

File: docs/cse/administration/custom-inventory-sources.md [hunk 3]
```diff
@@ -259,23 +252,25 @@ The table below lists attributes most typically used in user inventory records.
...
-| json field _raw "event.computer.deviceName as hostname \
-| json field _raw "event.computer.deviceName as hostname \
-| json field _raw "event.computer.ipAddress as ip \
-| json field _raw "event.computer.macAddress as mac \
-| json field _raw "event.computer.username as username \
-| json field _raw "event.computer.emailAddress as email \
-| json field _raw "event.computer.position as role \
-| where !(isEmpty(username)) \
-| count by os_version, hostname, ip, mac, username,email, role`
+```
```
Comment (kimsauce) [inline docs/cse/administration/custom-inventory-sources.md:255]: Can you push json here after the first three backticks(all lowercase)?

**Review by kimsauce — CHANGES_REQUESTED** [review]
1 small change

**Review by kimsauce — CHANGES_REQUESTED** [review]
Sorry, also line 280 needs json after backticks

**Review by kimsauce — APPROVED** [review]

**Pull request merged**